converters/fromZigbee.js

```diff
@@ -6521,6 +6521,23 @@ const converters = {
             };
         },
     },
+    aqara_knob_rotation: {
+        cluster: 'aqaraOpple',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.hasOwnProperty(570)) {
+                const act = {1: 'start_rotation', 2: 'rotation', 3: 'stop_rotation'};
```

Comment:
`start_rotation` -> maybe `start_rotating` is better?

Reply:
renamed start and end